sqlglot/expressions.py

```diff
@@ -6395,6 +6395,11 @@ class JSONTable(Func):
     }
 
 
+# https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions#json_type
+class JSONType(Func):
+    _sql_names = ["JSON_TYPE"]
```

Comment:
Have you checked whether `JSON_TYPE` is supported by other dialects and if it always expects exactly one argument?